pages/search.js

```diff
@@ -93,10 +99,8 @@ export default function Search({
       setNotFound(`${username} not found`);
     }
   }, [username]);
-  useEffect(() => {
-    if (searchInputRef.current) {
-      searchInputRef.current.focus();
-    }
+  useLayoutEffect(() => {
+    searchInputRef.current?.focus();
```

Comment:
```suggestion
    searchInputRef?.current.focus();
```
i think this is the correct way

Reply:
`searchInputRef` is all ready exist, why do need check for this I'm confused

Reply:
we are checking if the current exits or not